rj_gameplay/rj_gameplay/skill/shoot.py

```diff
@@ -10,32 +10,42 @@
 import stp.role as role
 import stp.action as action
 from rj_gameplay.action import pivot, kick
-from stp.skill.action_behavior import ActionBehavior
+from stp.skill.action_behavior import ActionBehavior, RobotActions
+from stp.skill.rj_sequence import RjSequence as Sequence
 import stp.rc as rc
-import stp.skill.sequence as sequence
+import numpy as np
 
 class IShoot(skill.ISkill, ABC):
     ...
 
 """
 A shoot skill which aims at the goal and shoots
 """
+random_shoot = random.uniform(-0.5,0.5)
```

Comment:
```suggestion
random_shoot = np.random.uniform(-0.5,0.5)
```

Sim doesn't launch without this